Sources/CustomHTTPProtocol.swift

```diff
@@ -62,17 +62,7 @@ public class CustomHTTPProtocol: URLProtocol {
     }
     
     private func body(from request: URLRequest) -> Data? {
-        return request.httpBody ?? request.httpBodyStream.flatMap { stream in
-            let data = NSMutableData()
-            stream.open()
-            while stream.hasBytesAvailable {
-                var buffer = [UInt8](repeating: 0, count: 1024)
-                let length = stream.read(&buffer, maxLength: buffer.count)
-                data.append(buffer, length: length)
-            }
-            stream.close()
-            return data as Data
-        }
+        return request.httpBody
```

Comment:
This sounds good to me, at least for the moment since it's better to not show the body rather than send the app in an infinite loop.
I opened a related issue here https://github.com/pmusolino/Wormholy/issues/93